fendermint/app/src/cmd/run.rs

```diff
@@ -237,3 +250,11 @@ fn to_resolver_config(settings: &Settings) -> anyhow::Result<ipc_ipld_resolver::
 
     Ok(config)
 }
+
+fn to_address(sk: &SecretKey, kind: &AccountKind) -> anyhow::Result<Address> {
```

Comment:
Why do you need to re-implement [here](https://github.com/consensus-shipyard/fendermint/pull/285/files#diff-7d11b1e9a3cc49dd08f1497ba1c3e20ccd2a8b82f5618a46df2e58eb8cb4f870R384)?

Reply:
Because these are two distinct `AccountKind` types at the moment: 
* one in the `fendermint_app_settings` which deals with deserializing the config file
* the other in `fendermint_app_options` which is about CLI parameters

These two crates have no reference to each other at the moment and sharing an `AccountKind` didn't seem important enough to introduce it. Not sure where else to put it at the moment. The one in `options` needs a `clap` specific attribute too, which is not something I would like to see spreading, although it might work with `serde` I don't know.